src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded [ingress-nginx to v1.11.2](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.11.2)  <!-- 8460 -->
```

Comment:
@codyjlandstrom the breaking change is a very uncommon case. I would explain more about it. Maybe something like "This `ingress-nginx` release comes with a breaking change only affecting edge case" And link to the issue where it's explained:
https://github.com/kubernetes/ingress-nginx/issues/11866
They didn't document it as a breaking change on their release notes